### DISCUSSION TITLE
1.9.0很期待,官网上暂时还没有手动部署过程

### DISCUSSION THREAD

**Discussion by ccl123456789012** [hhyo/Archery#1875] [General]
我看了 archery/settings.py 这个文件和1.8.5的差异有点大，不知道该怎么修改,望尽快更新

**Comment by hhyo** [comment]
如果不使用系统环境变量，仅新增这一个参数配置
https://github.com/hhyo/Archery/blob/ecc0bc773b158a3d35c869477ecadc19d0e01889/archery/settings.py#L39

**Comment by ccl123456789012** [comment]
1.我使用k8s部署，自己制作镜像
2.意思是说1.8.5的archery/settings.py
仅需要添加CSRF_TRUSTED_ORIGINS = env("CSRF_TRUSTED_ORIGINS") 
就可以把该文件放到1.9.0版本中使用

**Reply by tangguangliang** [reply]
hello，大佬，你的archery部署成功了吗，请教一下呢，我https配置有点问题
